plugins/modules/zos_copy.py

```diff
@@ -1243,7 +1354,7 @@ def copy_to_member(
             else:
                 self.fail_json(msg=msg, rc=rc, stdout=out, stderr=err)
 
-        return dest.replace("\\", "")
+        return dest.replace("\\", "").upper()
```

Comment:
Is anything actually being uppercased here?

Reply:
If this writes to a pdse member whose name is auto-generated, this makes sure the destination is properly capitalized before adding it to the argument return.

Reply:
My bad :) I read this as it was not a replace :)